public/javascripts/ood_shell.js

```diff
@@ -148,10 +100,24 @@ OodShell.prototype.onTerminalResize = function (columns, rows) {
   // React to size changes here.
   // Secure Shell pokes at NaCl, which eventually results in
   // some ioctls on the host.
-  this.socket.send(JSON.stringify({
-    resize: {
-      cols: columns,
-      rows: rows
-    }
-  }));
+  
+  // iff the socket is ready. race condition in FF where we try to resize before the socket is open
+  if(this.socket.readyState == 1){ 
+    this.socket.send(JSON.stringify({
+      resize: {
+        cols: columns,
+        rows: rows
+      }
+    }));
+  }else{
+      this.cachedCols = columns;
+      this.cachedRows = rows;
+  }
```

Comment:
Now we can just cache the values we tried to resize to when we're still connecting and use the onopen callback to resize to whatever we've cached last.